crates/viewer/re_space_view_spatial/src/ui_2d.rs

```diff
@@ -249,12 +246,6 @@ impl SpatialSpaceView2D {
 
         // ------------------------------------------------------------------------
```

Comment:
for later consideration: it was actually nice to have a context menu _inside_ the view.